### PR TITLE
Created Turkey open source collection.

### PR DESCRIPTION
**Pull request by kiliczsh**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Turkish software developers has created tools that would be nice and motivative.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by kiliczsh** [comment]
@nholden @MikeMcQuaid for your information

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Nice work so far!

File: collections/made-in-turkey/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ - fatih/vim-go
+ - ssg/streetcoder
+ - ahmetb/kubectx
+ - eserozvataf/laroux.js
+ - f/vue-wait
+
+
+display_name: Made in Turkey
+created_by: kiliczsh
+image: made-in-turkey.png
```
[the remark this answers is not in the document]
Comment (kiliczsh) [reply collections/made-in-turkey/index.md:11]: Replaced with 320x213 version from [Wikipeadia](https://commons.wikimedia.org/wiki/File:Flag_of_Turkey.svg) @MikeMcQuaid

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @kiliczsh!

**Pull request merged**